server/sample/bookinfo.yaml

```
@@ -348,7 +348,7 @@ spec:
       targetPort: 9080
 
 ---
-apiVersion: extensions/v1beta1
+apiVersion: "networking.k8s.io/v1"
```

No only the `apiVersion` need to be changed. but also the rules structure. 
```suggestion
apiVersion: "networking.k8s.io/v1"
kind: Ingress
metadata:
  namespace: servicemesh
  labels:
    app.kubernetes.io/version: v1
    app.kubernetes.io/name: bookinfo
  name: bookinfo-ingress
spec:
  rules:
    - http:
        paths:
          - path: /
            backend:
              service:
                name: productpage
                port: 9080
      host: productpage.servicemesh.139.198.121.92.nip.io
```

---

update